### ISSUE TITLE
User Auth

### ISSUE THREAD

**Issue by kerapace** [kerapace/kinographist#2]
# User Auth
## Functionality
- [ ] The main button with logo should link to `/`
- [ ] Has working demo login
- [x] Smooth, bug free navigation
### Before Login
- [x] The `/login` page should not display a link to `Log In`. Same for `/signup`
- [x] Going to a random route `/#/oweiniouewbrviuwebv` should redirect or display a 404 page
- [x] Errors should display for both `/signup` and `/login`.
- [x] Errors should clear when moving between `/signup` and `/login`.
- [x] Can sign up a user
- [x] Can sign in as a user
- [x] Can log out a user
- [x] Can't sign up with the same username/email
- [x] Pressing enter after filling out the session form should use the form data, not the demo user
### After Login
- [x] Should not be able to visit `/login` or `/signup`
- [x] Should be able to refresh the page and still be logged in
## Style
- [x] Things should be positioned correctly.
- [x] All relevant elements should have `cursor: pointer` on hover
- [x] Remove Redux logger and all console.log()'s from production
## Seeds
- [x] Adequate and appropriate seeds

**Comment by samwalker191** [comment]
Looks good so far!

- [ ] Demo login returned errors of invalid credentials. You might have to seed your production database with the demo user.
- [ ] See above checklist for additional stuff.
- [ ] Styling is a good start. Make sure to work on it over the weekend to clean it up.

**Comment by kerapace** [comment]
Yeah, I made a demo login in development and forgot to put it in the seed. Thanks for noticing that one, I'll fix it for the next feature push.
